airbyte-integrations/connectors/source-file/unit_tests/test_source.py

```diff
@@ -149,12 +149,11 @@ def test_discover(source, config, client):
 
 def test_check_wrong_reader_options(source, config):
     config["reader_options"] = '{encoding":"utf_16"}'
-    with pytest.raises(Exception):
```

Comment:
Do you understand how this used to work? I do not :confused:

Reply:
We don't need this code there at all, we don't expect check raises any exceptions on fails, but fail message itself.